src/Shouldly/Shouldly.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>net451;net452;netstandard1.3</TargetFrameworks>
```

Comment:
Agreed, I presume this was temporary in an effort to troubleshoot the ci build.

@JosephWoodward I've PRed a working build. If you are happy to merge it would be good to get some more eyes on this.